formalization_of_commits.py

```diff
@@ -0,0 +1,66 @@
+import requests

```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 Delete generate_md.py 
 Mistakes: 
 Dude...it is not even in right format.